tests/testJointcal.py

```diff
@@ -75,8 +75,9 @@ def setUp(self):
         # position of the Twinkles run 1 catalog
         center = coord.IcrsCoord(53.00914*geom.degrees, -27.43895*geom.degrees)
         r = geom.Angle(3, geom.degrees)
-        astrometryTask = astrometry.AstrometryTask()
-        reference = astrometryTask.refObjLoader.loadSkyCircle(center, r, filterName='r')
+        config = astrom.LoadAstrometryNetObjectsTask.ConfigClass()
+        refLoader = astrom.LoadAstrometryNetObjectsTask(config=config)
+        reference = refLoader.loadSkyCircle(center, r, filterName='r')
         self.reference = reference.refCat.copy()  # for in-memory contiguity.
```

Comment:
Please don't use the same variable name (`reference`) to refer to two completely different kinds of objects.